src/app/shared/services/record.service.ts

```diff
@@ -41,4 +45,12 @@ export class RecordService {
       .map(res => res.json());
   }
 
+  saveRecord(record: Object): Observable<Object> {
+    let postData = {
+      'pid_type': this.type,
+      'recid': this.recid,
+      'json_data': record
+    };
+    return this.http.post('http://localhost:5000/editor/save', postData).map(res => res.json());
```

Comment:
After the commit I just merged https://github.com/inspirehep/record-editor/commit/3babfaa653790f58e06179870ce9804082412d06 we have a config object were we can add API endpoints and so on.

Then to use it gets injected [like this](https://github.com/inspirehep/record-editor/blob/3babfaa653790f58e06179870ce9804082412d06/src/app/editor-container/editor-container.component.ts#L40)

For the base URL it might not be needed, just writing `/editor/save` would be ok (although indeed that part could be added to a config const)